CMakeLists.txt

```diff
@@ -124,6 +125,19 @@ else()
   message(STATUS "  [CMake is tracking Git commits and branching. To disable use '-D GIT_TRACKING=OFF'.]")
 endif()
 
+# Additional versioning information for packaging
```

Comment:
Its a bit hard to read the the code and imagine what it does.
Can you document this a bit more extensively?
and give an example of the result in either case.

Reply:
I can. Basically: If it is not built from a release branch it will add the branch name and the last change date to everything that can handle strings in the version (eg. installer filenames).